### PR TITLE
[PowerPoint] (tags) work with tags

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
[PowerPoint] (tags) work with tags

### CODE REVIEW

File: docs/powerpoint/tagging-presentations-slides-shapes.md
Comment (alison-mk) [inline docs/powerpoint/tagging-presentations-slides-shapes.md:84]: In a team meeting recently, we decided: "The `Promise` object should be capitalized and in code ticks. The concept of a promise should be lower case." 
```suggestion
    - The call to `getSelectedDataAsync` is embedded in a promise-returning function. For more information about why and how to do this, see [Wrap Common APIs in promise-returning functions](../develop/asynchronous-programming-in-office-add-ins.md#wrap-common-apis-in-promise-returning-functions).
```

File: docs/powerpoint/tagging-presentations-slides-shapes.md
Comment (alison-mk) [inline docs/powerpoint/tagging-presentations-slides-shapes.md:136]: A suggestion to avoid "CustomPropertys": 
```suggestion
Add-ins can also apply tags to the presentation as a whole. This enables you to use tags for document-level metadata similar to how the [CustomProperty](/javascript/api/word/word.customproperty)class is used in Word. But unlike the Word `CustomProperty` class, the value of a PowerPoint tag can only be of type `string`.
```

File: docs/powerpoint/tagging-presentations-slides-shapes.md
Comment (alison-mk) [inline docs/powerpoint/tagging-presentations-slides-shapes.md:22]: The "Each type of parent object..." sentence is unclear to me. Are there parent objects in addition to Presentation, Slide, and Shape? A potential clarification: 
```suggestion
A tag is a key-value pair, where the value is always of type `string` and is represented by a [Tag](/javascript/api/powerpoint/powerpoint.tag) object. Each type of parent object, such as a [Presentation](/javascript/api/powerpoint/powerpoint.presentation), [Slide](/javascript/api/powerpoint/powerpoint.slide), or [Shape](/javascript/api/powerpoint/powerpoint.shape) object, has a `tags` property of type [TagsCollection](/javascript/api/powerpoint/powerpoint.tagcollection).
```

File: docs/powerpoint/tagging-presentations-slides-shapes.md
Comment (alison-mk) [inline docs/powerpoint/tagging-presentations-slides-shapes.md:28]: To reiterate, in case someone skips to this "Adding" section without reading the preceding paragraph. 
```suggestion
- The first parameter of the `add` method is the key in the key-value pair. 
```

File: docs/powerpoint/tagging-presentations-slides-shapes.md
Comment (alison-mk) [inline docs/powerpoint/tagging-presentations-slides-shapes.md:86]:
```suggestion
    - The `index` value of the slide is the 1-based value the user sees beside the slide in the PowerPoint UI thumbnails pane.
```

**Review by alison-mk — APPROVED** [review]
Approving with a couple suggestions. Since this PR adds a new article, I also suggest getting a review from our editor @lindalu-MSFT. 
Note: asked one question about this PR offline via email.

**Review by lindalu-MSFT — APPROVED** [review]
Suggested changes per MSFT style guidelines:
1. Imperative verb form in headings
2. Parallel sentence structure in lists

**Pull request merged**
